clients/privacy-center/pages/consent.tsx

```diff
@@ -108,13 +110,13 @@ const Consent: NextPage = () => {
    * ensures the browser's behavior matches what the server expects.
    */
   useEffect(() => {
-    setConsentCookie(
-      makeCookieKeyConsent({
-        consentOptions,
-        fidesKeyToConsent: persistedFidesKeyToConsent,
-        consentContext,
-      })
-    );
+    const cookie: FidesCookie = getOrCreateFidesCookie();
```

Comment:
Previously, this would "set or create" cookie, since it could safely just ignore the cookie values that were there. Now, I have this code just update the _consent_ section of the cookie, since that's what is being edited here.

This just means that the identity/meta sections aren't changed from the consent callback. Basically being defensive against future versions where we likely edit different parts of the cookie in different places.